docs/REST API/rest-api.md

```diff
@@ -4735,33 +4718,48 @@ Content-Type: application/octet-stream
 
 `Status: 200`
 
-### Get metadata about an EULA file
+### Get or download an EULA file
 
 _Available in Fleet Premium_
 
-Get information about the EULA file that was uploaded to Fleet. If no EULA was previously uploaded, this endpoint returns a `404` status code.
+`GET /mdm/eula/{team_id}`
 
-`GET /api/v1/fleet/mdm/apple/setup/eula/metadata`
+#### Parameters
 
-#### Example
+| Name       | Type    | In    | Description                                      |
+| -----------| ------  | ----- | ------------------------------------------------ |
+| team_id    | integer | path  | **Required.** The desired EULA file's team's ID. |
+| alt        | string  | query | If specified and set to "media", downloads the EULA file's contents. |
```

Comment:
```suggestion
| alt        | string  | query | If specified and set to "media", downloads the EULA file. |
```
Same as above. I would cut "...'s contents", since it's easier to read without.